defaults/main.yml

```diff
@@ -79,3 +79,4 @@ mssql_ad_kerberos_password: >-
 
 mssql_run_selinux_confined: "{{ __mssql_confined_supported | d(false) }}"
 mssql_manage_selinux: false
+mssql_manage_tuned: "{{ __mssql_tuned_supported | d(false) }}"
```

Comment:
Please move this to `vars/main.yml` and rename it to `__mssql_manage_tuned`. `defaults/main.yml` is for variables that the role expects users might set, while `mssql_manage_tuned` is more like an internal variable. Prefixing a variable with two underscores shows that it is an internal variable.